File: DDD/ControleLivros_Unity/ArtigoUnity.Infraestrutura/EF/Repositorio/Implementacao/EditoraRepositorio.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ArtigoUnity.Dominio.Entidade;
using ArtigoUnity.Dominio.Repositorio.Interfaces;
using ArtigoUnity.Infraestrutura.EF.ContextoBD;

namespace ArtigoUnity.Infraestrutura.EF.Repositorio.Implementacao
{
    public class EditoraRepositorio: RepositorioBase<Editora,ContextoDb>, IEditoraRepositorio
    {
        public EditoraRepositorio(ContextoDb contexto)
            :base(contexto)
        {

        }
    }
}

File: DDD/Java/Sistema de Gerencia de Obras/editora/src/main/java/br/com/simasoft/editora/dominio/contrato/EditoraRepositorio.java
package br.com.simasoft.editora.dominio.contrato;

import br.com.simasoft.editora.dominio.entidade.Editora;
import br.com.simasoft.editora.infraestrutura.persistencia.hibernate.comum.HibernateRepositorioGenerico;

public interface EditoraRepositorio extends HibernateRepositorioGenerico<Editora, Long> {

}
